lib/include/hse_ikvdb/ikvdb.h

```diff
@@ -364,7 +364,7 @@ ikvdb_kvs_count(struct ikvdb *kvdb, unsigned int *count);
  * @params:    KVS cparams
  */
 merr_t
-ikvdb_kvs_create(struct ikvdb *kvdb, const char *kvs_name, const struct kvs_cparams *params);
+ikvdb_kvs_create(struct ikvdb *kvdb, const char *kvs_name, struct kvs_cparams *params);
```

Comment:
HSE_REVISIT: convert last arg back to const

Reply:
done